templates/shared_services/airlock_notifier/template_schema.json

```diff
@@ -4,7 +4,7 @@
   "type": "object",
   "title": "Airlock Notifier Shared Service",
   "description": "A shared service notifying on Airlock operations",
-  "required": ["smtp_server_address", "smtp_username", "smtpPassword", "smtp_from_email"],
+  "required": ["smtp_server_address", "smtp_server_port", "smtp_username", "smtpPassword", "smtp_from_email"],
```

Comment:
If we add a default, can the port still be optional?

Reply:
Yes, it can.